zips/zip-0203.rst

```diff
@@ -31,12 +31,12 @@ be mined, and potentially simplifying bidirectional payment channels by reducing
 to store and compress revocations for past states, since transactions not committed to the
 chain could expire and become invalid after a period of time.
 
-If the expiry is at block height $N$, then the transaction must be included in block
+If the expiry is at block height $N\!$, then the transaction must be included in block
 $N$ or earlier. Block $N+1$ will be too late, and the transaction will be
 removed from the mempool.
 
 The new consensus rule will enforce that the transaction will not be considered valid if
-included in block of height greater than $N$, and blocks that include expired
+included in block of height greater than $N\!$, and blocks that include expired
```

Comment:
Why the additions of `\!` here? Elsewhere in this commit I've seen a bunch of removals, and I'm not sure what the change means.

Reply:
The `render.sh` script will automatically add one `\!` in this case. However, because of the shape of $N$ (italic uppercase with a long stroke on the right), two looks better.